requirements/edx/base.in

```diff
@@ -35,7 +35,7 @@ ddt==0.8.0                          # via xblock-drag-and-drop-v2 (which probabl
 defusedxml==0.4.1                   # XML bomb protection for common XML parsers
 Django==1.11.14                     # Web application framework
 django-babel-underscore             # underscore template extractor for django-babel (internationalization utilities)
-django-config-models==0.1.8         # Configuration models for Django allowing config management with auditing
+django-config-models                # Configuration models for Django allowing config management with auditing
```

Comment:
This unpinning always feels good - like progress. One less thing that will bit-rot over time.